nodes.py

```diff
@@ -58,6 +58,30 @@ def INPUT_TYPES(s):
     def combine(self, conditioning_1, conditioning_2):
         return (conditioning_1 + conditioning_2, )
 
+class ConditioningAddWeighted:
+    @classmethod
+    def INPUT_TYPES(s):
+        return {"required": {"conditioning_from": ("CONDITIONING", ), "conditioning_to": ("CONDITIONING", ),
+                              "conditioning_from_strength": ("FLOAT", {"default": 1.0, "min": 0.0, "max": 10.0, "step": 0.1}),
+                              "conditioning_to_strength": ("FLOAT", {"default": 0.0, "min": 0.0, "max": 10.0, "step": 0.1})
+                             }}
+    RETURN_TYPES = ("CONDITIONING",)
+    FUNCTION = "addWeighted"
+
+    CATEGORY = "conditioning"
+
+
+    def addWeighted(self, conditioning_from, conditioning_to, conditioning_from_strength, conditioning_to_strength):
+        conditioning_from_tensor = conditioning_from[0][0]
+        conditioning_to_tensor = conditioning_to[0][0]
```

Comment:
I think the better behaviour would be to apply it to every cond in the to list. Just copy this loop: https://github.com/comfyanonymous/ComfyUI/blob/master/nodes.py#L377